CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# TBD
+
+## Enhancements
+
+- Add assertions for and automatic verification of Bugsnag-Integrity headers
```

Comment:
```suggestion
- Add assertions for automatic verification of Bugsnag-Integrity headers
```

Reply:
Ah, what I really meant was `Add assertions for, and automatic verification of, Bugsnag-Integrity headers`(with commas), because there are both opt-in Cucumber steps and the no-option, check every request I get feature that have been added.  I'll reword to make that clearer.